test/smoke/src/helpers/setupEnvironmentHelper.ts

```diff
@@ -91,6 +90,46 @@ export function prepareReactNativeApplication(workspaceFilePath: string, resourc
     fs.writeFileSync(path.join(vsCodeConfigPath, "launch.json"), fs.readFileSync(launchConfigFile));
 }
 
+export function installExtensionVSIX(extensionDir: string, testVSCodeExecutablePath: string, resourcesPath: string, isInsiders: boolean) {
+    let args: string[] = [];
+    args.push(`--extensions-dir=${extensionDir}`);
+    let dirFiles = fs.readdirSync(resourcesPath);
+    let extensionFile = dirFiles.find((elem) => {
+        return elem.match(/.*\.(vsix)/);
+    });
+    if (!extensionFile) throw new Error(`React Native extension .vsix is not found in ${resourcesPath}`);
+
+    extensionFile = path.join(resourcesPath, extensionFile);
+    args.push(`--install-extension=${extensionFile}`);
+    let codeExecutableScript = isInsiders ? "code-insiders" : "code";
+    testVSCodeExecutablePath = path.join(testVSCodeExecutablePath, codeExecutableScript);
+    if (process.platform === "win32") {
+        testVSCodeExecutablePath += ".cmd";
+    }
+    console.log(`*** Installing ${extensionFile} into ${extensionDir} with ${testVSCodeExecutablePath} executable`);
+    cp.spawnSync(testVSCodeExecutablePath, args, {stdio: "inherit"});
+    console.log(`*** Deleting ${extensionFile} after installation`);
+    rimraf.sync(extensionFile);
+}
+
+export function runAndroidEmulator() {
+    if (!process.env.ANDROID_EMULATOR) throw new Error("Environment variable 'ANDROID_EMULATOR' is not set. Exiting...");
+    terminateAndroidEmulator();
+    console.log(`*** Executing Android emulator with 'emulator -avd ${process.env.ANDROID_EMULATOR}' command...`);
+    cp.spawn("emulator", ["-avd", process.env.ANDROID_EMULATOR || "", "-wipe-data"], {stdio: "inherit"});
+
+}
+
+export function terminateAndroidEmulator() {
```

Comment:
Please, add comment to the function, that it is expected that only one emulator is being terminated.

Reply:
Added!